corehq/apps/cloudcare/templates/formplayer/debugger.html

```diff
@@ -89,18 +89,34 @@
     <!--ko if: xPathQuery.errorResult()-->
     <div data-bind="text: xPathQuery.errorResult(), css: 'text-danger'"></div>
     <!--/ko-->
-    <!--ko if: options.useCodeMirror && xPathQuery.successResult()-->
+    <!--ko if: xPathQuery.isSuccessTruncated() && options.showInFull-->
+    <div>
+        <i>Showing the first </i><i data-bind="text: xPathQuery.getMaxLines()"/><i> lines...</i>
+        <a class="clickable pull-right" data-clipboard-action="copy" data-bind="clipboardButton: xPathQuery.getFullSuccessResult()">
+            Copy Full Evaluation to Clipboard
+        </a>
+    </div>
+    <!--/ko-->
+    <!--ko if: options.showInFull && xPathQuery.successResult()-->
     <textarea type="text" data-bind="
         codeMirror: xPathQuery.successResult()
     "></textarea>
     <!--/ko-->
-    <!--ko if: options.useCodeMirror && xPathQuery.hasTrace()-->
+    <!--ko if: xPathQuery.isTraceTruncated() && options.showInFull-->
+    <div>
+        <i>Showing the first </i><i data-bind="text: xPathQuery.getMaxLines()"/><i> lines...</i>
+        <a class="clickable pull-right" data-clipboard-action="copy" data-bind="clipboardButton: xPathQuery.getFullTraceResult()">
```

Comment:
`clickable` is a confusing name for this. I think `clipboard` or something similar might be a better name.